src/global.scss

```diff
@@ -326,4 +326,9 @@ ion-searchbar {
           padding: 5px 14px 5px 45px !important;
       }
   }
+}
+
+::-webkit-scrollbar {
```

Comment:
Does it affect all the scroll bars in the portal

Reply:
no bro other page like home page, mentor directory we are able to see the scrolls

Reply:
No only in side menu. i am able to see the scrolls in home page and mentor directory